packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -364,7 +368,11 @@ test('first', async t => {
     undefined,
     500n,
   );
-  const { vaultFactory, lender, aethVaultManager } = services.vaultFactory;
+  const {
+    vaultFactoryCreator: vaultFactory,
```

Comment:
it would be good to not have this property renaming.

Reply:
goal was to not have `vaultFactory.vaultFactory` as before. granted the type system indicates they're different, but just eyeballing it's not clear.

In general I think we would benefit from more consistency in how we name facets, contracts, and virtual/durable objects in contracts. That's a bigger endeavor that is out of scope so I'll revert this change.